### PR TITLE
PR: Update README and example

### LINKED ISSUE #60: Provide defaults example
GitHub Actions added a defaults feature that seems almost custom made for setup-miniconda; see https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_iddefaultsrun

With that, you can set the default run shell to "bash -l {0}", for example, for all run steps. Using this in one of the examples in the readme would be helpful, I believe.

### PR DESCRIPTION
**Pull request by goanpeca**
Fixes #60
Related to #92

Added this new section on the README and updated the examples accordingly.

Screen Shot 2020-11-16 at 14 28 52

### CODE REVIEW

**Comment by goanpeca** [comment]
Pinging @henryiii :)

File: README.md [hunk 1]
```diff
@@ -364,6 +362,37 @@ you may want to
...
 
+## Use a default shell
+
+Assuming you are using the bash shell, now adding to `shell: bash -l {0}` to every single step can
+be avoided if your workflow uses the same shell for all the steps.
+
+By adding a `defaults` section and specifying the `bash -l {0}`, all steps in the job will default
+to that value.
+
+For other shells, make sure to use the right `shell` parameter as the default value.
```
Comment (jaimergp) [inline README.md:373]: I'd add a little reference to the note in the section below. Something like: 

```suggestion
For other shells, make sure to use the right `shell` parameter as the default value. Check the [section below](#important) for some examples.
```

**Review by jaimergp — APPROVED** [review]

**Comment by goanpeca** [comment]
Thanks @jaimergp !

**Comment by henryiii** [comment]
A little late, but looks good to me!

**Comment by goanpeca** [comment]
>A little late, but looks good to me!

Awesome :) !

**Pull request merged**